doc/guides/collaborator-guide.md

```diff
@@ -244,6 +244,12 @@ Copy/paste the URL for the job into a comment in the pull request.
 [`node-test-pull-request`](https://ci.nodejs.org/job/node-test-pull-request/)
 is an exception where the GitHub bot will automatically post for you.
 
+The [`node-test-pull-request`](https://ci.nodejs.org/job/node-test-pull-request/)
+CI job can be started via label, by adding the `request-ci` label in
+the pull request, and the `github-actions bot` will start
+the `node-test-pull-request` automatically, in case the action fails,
```

Comment:
This should likely be split into multiple sentences:


> The [`node-test-pull-request`](https://ci.nodejs.org/job/node-test-pull-request/) CI job can be started via label by adding the `request-ci` label to the pull request. Once this label is added, `github-actions bot` will start the `node-test-pull-request` automatically. Should there be a failure, `github-actions bot` will update the label with `request-ci-failed`.

Reply:
> Should there be a failure, `github-actions bot` will update the label with request-ci-failed.

I think this needs some rephrasing so folks don't assume the label will be added if the Jenkins job fails. What about:

> If the `github-actions bot` is unable to start the job, it will update the label with `request-ci-failed`

Reply:
Thanks for reviewing, I've made the corresponding fixes in this commit: 35ba804d1eca44a99c542837d51d800e3e2e4979